.github/renovate.json

```diff
@@ -1,6 +1,10 @@
 {
   "extends": [
-    "config:base"
-  ],
-  "enabledManagers": ["gradle-wrapper"]
+    "config:base",
```

Comment:
Does it need all these other extends if its extending the sscs-renovate.json with those in ?

Reply:
Hmm probably the documentation is not very clear on this

Reply:
I'll try without